hack/tools/release/notes/main.go

```diff
@@ -103,10 +103,16 @@ func (cmd *notesCmd) run() error {
 	from, to := parseRef(cmd.config.fromRef), parseRef(cmd.config.toRef)
 
 	printer := newReleaseNotesPrinter(cmd.config.repo, from.value)
-	printer.isPreRelease = cmd.config.preReleaseVersion
 	printer.printDeprecation = cmd.config.deprecation
 	printer.printKubernetesSupport = cmd.config.addKubernetesVersionSupport
 
+	if cmd.config.releaseType != "" {
```

Comment:
super nit: It might be more readable to make this a switch statement rather than a nested if

Reply:
Aso thought about adding this to a separate func, but this reduces readability, tbh :)

But open for other ideas